doc/Makefile-broken

```diff
@@ -0,0 +1,178 @@
+# Makefile for Sphinx documentation
```

Comment:
This filename is confusing -- if it's truly broken we should delete it; if it's needed for mvn, as the comments imply, I'd suggest renaming it to something like Makefile-for-mvn.